exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py

```diff
@@ -356,7 +375,9 @@ def help(self) -> None:
         self.warning("  --/exts/ansys.tools.omniverse.core/help=1")
         self.warning("     Display this help.")
         self.warning("  --/exts/ansys.tools.omniverse.core/run=1")
-        self.warning("     Run the server.")
+        self.warning("     Run the server until the DSG connection is broken.")

```

Comment:
Do the old style options, with --/exts/... remain this way, for the kit extension?  And is that so we can continue to have an Omniverse GUI plugin?  

Or is this code that just hasn't been updated?

Reply:
@david-bremer this PR breaks the extensions for now. Not too worried about this as OV will not see the change until we formally release again.   Yes, I need to replace that code, but I'd like to do so in another PR (as I want to get EnSight builds running against the rest of this code quickly).